docs/computer_setup.md

```diff
@@ -2,82 +2,161 @@
 How to set up your system, environment, and upload file(s) into the database
 
 ## Setting up operating system
-Make sure you have an up-to-date web browser. Follow the directions [here](https://docs.microsoft.com/en-us/windows/wsl/install) to set up a bash shell, git, and python 3 (please set up all three). (add miniconda)
-* Run as administrator 
-	*  If using cmd, right-click on the program and click “Run as Administrator”
-* Skip distribution
+Make sure you have an up-to-date web browser. Follow the directions
+[here](https://docs.microsoft.com/en-us/windows/wsl/install) to set up a bash shell,
+git, and python 3. (add miniconda)
+- Run as administrator 
+	-  If using cmd, right-click on the program and click “Run as Administrator”
+- Skip distribution
 
 Here is some basic info about each of the programs: 
-* Bash shell
-	The bash shell is the interface between the user (you) and the operating system. It is a command line interface shell program extensively used in Linux and macOs operating systems. Bash is used when users want to control their computer without having to navigate menus, options, and windows while interacting in your computer. For example, if you want to quickly create, edit, or delete multiple files, it’s easier to use Bash instead of finding each file by pointing and clicking on multiple directories.
+- Bash shell
```

Comment:
Consider defining/elucidating the relationship between the terms 'shell', 'command line', and 'terminal'.